openpathsampling/storage/stores/collectivevariable.py

```diff
@@ -150,18 +150,4 @@ def cache_all(self):
         Fill the caches of all CVs
 
         """
-        map(self.fill_cache, self)
-
-    def fill_cache(self, cv):
-        """
-        Fill the cache of a specific CV
-
-        Parameters
-        ----------
-        cv : :obj:`openpathsampling.CollectiveVariable`
-            the CV from which you want the cache to be filled
-
-        """
-        store = self.storage.snapshots.cv_list.get(cv)
-        if store is not None:
-            store.fill_cache()
+        pass
```

Comment:
??? empty function? Is this now obsolete, but kept in for API compatibility? If so, please add deprecation note.